versioned_docs/version-0.5.0/kubeslice-cli/getting-started.mdx

```diff
@@ -702,7 +702,7 @@ Use this command to delete the kind clusters used for KubeSlice demonstration.
 ### Syntax
 
 ```
-kubeslice-cli <uninstall|cleanup>
+kubeslice-cli <uninstall|cleanup> [options]
```

Comment:
Modify the description of the uninstall caommand

Reply:
updated